index.html

```diff
@@ -6414,11 +6443,20 @@ <h3>Issuer</h3>
         </p>
 
         <p>
-Relevant metadata about the <code>issuer</code> <a>property</a> is expected
-to be available to the <a>verifier</a>. For example, an <a>issuer</a> can
-publish information containing the public keys it uses to digitally sign
-<a>verifiable credentials</a> that it issued. This metadata is relevant when
-checking the proofs on the <a>verifiable credentials</a>.
+Relevant metadata that is related to the <code>issuer</code> <a>property</a>
+is available to the <a>verifier</a> through the
+<a href="#verify-securing-mechanism">securing mechanism verification
+algorithm</a> as defined in Section <a href="#securing-mechanisms"></a>. This
+information includes the controller, which is typically the `issuer`, of the
```

Comment:
```suggestion
information includes the verified controller, which is typically the `issuer`, of the
```

Reply:
@awoie wrote:
> I assume the securing mechanism verifies the returned controller is the controller of the securing mechanism?

Yes, correct. If that's not done, you can't trust the securing mechanism. 

Merging your change suggestion.